nevergrad/optimization/differentialevolution.py

```diff
@@ -313,12 +325,15 @@ def __init__(
 
 
 DE = DifferentialEvolution().set_name("DE", register=True)
+HSDE = DifferentialEvolution(high_speed=True).set_name("HSDE", register=True)
```

Comment:
as usual, is it useful as non-experimental variant? we should avoid cluttering the namespace

Reply:
I would rather keep it here if positive results are confirmed a little bit more.
Still I agree that a big cleaning of optimizerlib.py is necessary; based on splitting (a lot) and removing (a bit).

Reply:
> if positive results are confirmed a little bit mor

My recommendation would be, as long as it's not confirmed, put it into experimentalvariants, and as soon as it is, move it here. Here should not be the defaults, the default should be experimentalvariants because removing stuff from experimentalvariants is not a breaking change